### ISSUE TITLE
Error: Unknown rule 'preferDouble'. Did you mean 'preferKeyPath'?

### ISSUE THREAD

**Issue by verodanilova** [nicklockwood/SwiftFormat#1316]
Hey!

After updating to v. 0.50.4 command-line tool fails with the following error:
`error: Unknown rule 'preferDouble'. Did you mean 'preferKeyPath'?`

Our .swiftformat file:
```
--exclude **/Pods,vendor,**/*generated.swift

# Options
--allman false
--elseposition nextline
--empty void
--ifdef no-indent
--indent 4
--hexliteralcase lowercase
--nospaceoperators
--semicolons never
--stripunusedargs closure-only
--swiftversion 5
--trimwhitespace always
--xcodeindentation enabled

# Rules
--rules anyObjectProtocol,assertionFailures,blankLinesAroundMark,blankLinesBetweenScopes,blankLinesBetweenImports,braces,consecutiveBlankLines,consecutiveSpaces,duplicateImports,emptyBraces,indent,initCoderUnavailable,leadingDelimiters,linebreakAtEndOfFile,numberFormatting,preferDouble,preferKeyPath,redundantBackticks,redundantBreak,redundantClosure,redundantExtensionACL,redundantFileprivate,redundantGet,redundantInit,redundantLet,redundantLetError,redundantNilInit,redundantObjc,redundantParens,redundantPattern,redundantRawValues,redundantReturn,redundantVoidReturnType,semicolons,spaceAroundBraces,spaceAroundBrackets,spaceAroundComments,spaceAroundGenerics,spaceAroundParens,spaceAroundOperators,spaceInsideBraces,spaceInsideBrackets,spaceInsideComments,spaceInsideGenerics,spaceInsideParens,trailingClosures,trailingCommas,trailingSpace,typeSugar,unusedArguments,void,yodaConditions,modifierOrder,elseOnSameLine,
```

Could you please help to check, what's wrong? 🙏 

We use XCode 14.0, CocoaPods 1.11.3

**Comment by nicklockwood** [comment]
@verodanilova it looks like the binary that I shipped with the 0.50.4 update was the wrong version 🤦‍♂️ that rule is scheduled to be removed, but not until the next major update. Sorry about that.

**Comment by nicklockwood** [comment]
@verodanilova This should now be fixed in 0.50.5

**Comment by verodanilova** [comment]
Thank you, @nicklockwood ! 🙏
